Themis/Views/Assessment/CorrectionSidebar/EditFeedback/AddFeedbackView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AddFeedbackView.swift
+//  Themis
+//
+//  Created by Paul Schwind on 14.12.22.
+//
+
+import SwiftUI
+
+struct AddFeedbackView: View {
+    @Binding var assessmentResult: AssessmentResult
+    @ObservedObject var cvm: CodeEditorViewModel
+    let type: FeedbackType
+
+    @Binding var showSheet: Bool
+
+    @State var feedback = AssessmentFeedback(credits: 0.0, type: .general)
+
+    func initFeedback() {
+        feedback = AssessmentFeedback(credits: 0.0, type: type)
+    }
+
+    var body: some View {
+        EditFeedbackViewBase(
+            assessmentResult: $assessmentResult,
+            cvm: cvm,
+            feedback: $feedback,
+            showSheet: $showSheet,
+            title: "Add feedback",
+            edit: false,
+            type: type
+        )
+            .onAppear {
+                initFeedback()
+            }
+    }
+}
```

Comment:
I don't completely get the suggestion 🤔 
The problem is that we need to bind the feedback in the EditFeedbackView. Because of this, we cannot initialize it with an empty feedback there (binding expected). So if we would want to just use an EditFeedbackView to add a new feedback like before, we would have to initialize the feedback ourselves, which is not so nice. Therefore there now is an `AddFeedbackView` that does that and also manages the title using the abstraction instead of a conditional like before. Opinions?

Reply:
Hm yeah I don't see any problem here, I would have done it the same. I think it is fine